src/m365/cli/commands/config/config-set.ts

```diff
@@ -0,0 +1,77 @@
+import * as chalk from "chalk";
+import { Logger } from "../../../../cli";
+import { CommandOption } from "../../../../Command";
+import configstore from "../../../../configstore";
+import GlobalOptions from "../../../../GlobalOptions";
+import AnonymousCommand from "../../../base/AnonymousCommand";
+import commands from "../../commands";
+const Configstore = require('configstore');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  key: string;
+  value: string;
+}
+
+class CliConfigSetCommand extends AnonymousCommand {
+
+  public get name(): string {
+    return `${commands.CONFIG_SET}`;
+  }
+
+  public get description(): string {
+    return 'Manage global configuration settings about the CLI for Microsoft 365';
+  }
+
```

Comment:
It would be interesting to keep track in our telemetry which settings are being set so that we can better understand how people use CLI.

Reply:
That was my first though too, but in a long term perspective, shouldn't the command allow to specify user settings? And in that case, the telemetry wouldn't fit in, right?

Reply:
For each property that we allow to configure we can choose if we want to track its value (if it's a boolean on an enum) or just the fact that it's been set because it could contain a path, an ID or something else that's meaningless to aggregate and could contain PII.